scripts/ts-api-extractor/src/adapters/out/ts-morph/parsers/props/filter.ts

```
@@ -0,0 +1,93 @@
+/**
+ * Prop filtering module
+ *
+ * Determines which props to include/exclude from extraction output.
+ */
+import type { Symbol } from 'ts-morph';
```

Biome (2.4.7)

[error] Do not shadow the global "Symbol" property.

(lint/suspicious/noShadowRestrictedNames)